imports/packages/clipboard/clipboard.tsx

```diff
@@ -0,0 +1,68 @@
+import { DeepClient } from "@deep-foundation/deeplinks/imports/client";
+import { Clipboard } from '@capacitor/clipboard';
+
+export async function createClipboard({ deep, deviceLinkId }: { deep: DeepClient, deviceLinkId: any }) {
+    const { type, value } = await Clipboard.read();
+    const clipboardType = await deep.id("@deep-foundation/clipboard", "clipboard");
+
+    const ml = new ManagerLinks(deep);
+
+    const contactLink = await ml.createLink({
+        containName: "clipboard",
+        value: value,
+        contain_from_id: deviceLinkId,
+        type_id: clipboardType,
+    });
+}
+
+export async function createClipboardPackage({ deep, deviceLinkId }: { deep: DeepClient, deviceLinkId: any }) {
+    const typeTypeLinkId = await deep.id("@deep-foundation/core", "Type");
+    const packageTypeLinkId = await deep.id("@deep-foundation/core", "Package");
+
+    const ml = new ManagerLinks(deep);
+
+    const clipbloardPackage = await ml.createLink({
```

Comment:
Reduce amount of deep network requests 

_This comment applies to all the places working like these lines of code_